core/citrus-spring/src/main/java/org/citrusframework/reporter/ReporterConfig.java

```diff
@@ -31,4 +42,29 @@ public JUnitReporter junitReporter() {
     public TestReportersFactory testReporters() {
         return new TestReportersFactory();
     }
+
+    static class LoggingReporterEnablementCondition  implements Condition {
```

Comment:
eagle eye on duty! 😉 (you've got some copy-pasta for all the `Condition`s below...

```suggestion
    static class LoggingReporterEnablementCondition implements Condition {
```